polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -1551,10 +1553,8 @@ ValueCategory MLIRScanner::VisitCastExpr(CastExpr *E) {
             }
         }
     auto SE = Visit(E->getSubExpr());
-#ifdef DEBUG
     if (!SE.val)
```

Comment:
Naked trace. We need to place this one in a LLVM_DEBUG.

Reply:
Even if the code asserts later on one can always compile with traces enabled to get more information, so IMO it still make sense to use LLVM_DEBUG. Same comment for other similar changes. Also wrapping debug code in LLVM_DEBUG makes it easily identifiable to the human eye.

Reply:
I added the DEBUG macro... I guess I *should* have used LLVM_DEBUG... oh well.